data/hlint.yaml

```diff
@@ -474,6 +477,7 @@
 
     - warn: {lhs: fst (unzip x), rhs: map fst x}
     - warn: {lhs: snd (unzip x), rhs: map snd x}
+    - hint: {lhs: "\\(x,y) -> (x,f y)", rhs: fmap f}
```

Comment:
Not totally convinced  by this. I'd prefer `Bifunctor.second` over fmap, since that's clearer.

Reply:
Is it really clearer? It's dragging in Bifunctor, which is much less common/well-known than Functor.

Reply:
I'm inclined to say because neither is obviously clearer to leave it out entirely. I think the functor instance of `(,)` comes up when people are thinking tricky, whereas for Bifunctor the most common type is `(,)` by far.

In truth, I'd use https://hackage.haskell.org/package/extra-1.6.17/docs/Data-Tuple-Extra.html#v:second which is restricted to tuples - which I think is a good idea - but want to stick to the base library for hints.